### PR TITLE
29 MH3 scripts

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/waterlogged_teachings_inundated_archive.txt [hunk 1]
```diff
@@ -2,8 +2,6 @@ Name:Waterlogged Teachings
 ManaCost:3 UB
 Types:Instant
 A:SP$ ChangeZone | ChangeType$ Instant,Card.hasKeywordFlash | ChangeNum$ 1 | Origin$ Library | Destination$ Hand | Shuffle$ True | SpellDescription$ Search your library for an instant card or a card with flash, reveal it, put it into your hand, then shuffle.
-#TODO (copied from Mystical Teachings): The AI will currently search for the most expensive valid card in the library. This can be used to the AI's advantage with
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/waterlogged_teachings_inundated_archive.txt]: Don't think we put comments like this anymore (this is better as a github issue ?)

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/waterlogged_teachings_inundated_archive.txt]: indeed

File: forge-gui/res/cardsfolder/upcoming/skoa_embermage.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Skoa, Embermage
+ManaCost:4 R R
+Types:Legendary Creature Goblin Wizard
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDealDamage | TriggerDescription$ When CARDNAME enters the battlefield, it deals 4 damage to any target.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ 4
+A:AB$ DealDamage | Cost$ Discard<1/Card.namedSkoa, Embermage> Sac<2/Mountain> | CostDesc$ Grandeur — Discard another card named Skoa, Embermage, Sacrifice two Mountains: | ValidTgts$ Any | NumDmg$ 4 | SpellDescription$ NICKNAME deals 4 damage to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/skoa_embermage.txt:7]: @Simisays 
did you check if your change ; ➡️  , works?
The valid might get split badly with it now

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/skoa_embermage.txt:7]: I think so yeah 
image

The previous version didn't let me activate even though all the conditions where met

**Review by Simisays — APPROVED** [review]

**Review by Agetian — APPROVED** [review]

**Pull request merged**